pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -386,6 +386,18 @@ func (s *ServiceImpl) buildDashboardNavLinks(c *contextmodel.ReqContext) []*navt
 		})
 	}
 
+	//if s.features.IsEnabledGlobally(featuremgmt.FlagDashboardRestore) {
```

Comment:
This will be needed by the frontend. We could remove this commented code from this PR and add it in another one.